.github/workflows/integration.yaml

```diff
@@ -1,74 +1,184 @@
-name: Integration Tests For Retina
+name: Build images and run E2E tests. 
```

Comment:
File a Issue to remove duplicate code please.

Reply:
already field. https://github.com/microsoft/retina/issues/137